app/services/scrapers/ocso_warrants.rb

```diff
@@ -15,13 +14,13 @@ def perform
           case_number: warrant_hash[:case_number],
           bond_amount: warrant_hash[:bond_amount],
           issued: warrant_hash[:issued],
-          counts: warrant_hash[:counts]
+          counts: warrant_hash[:counts],
+          updated_at: DateTime.now
```

Comment:
```suggestion
          updated_at: DateTime.now,
          resolved_at: nil
```
This way you don't need to run any code to removed resolved_at (because it'll fix itself if it was flagged accidentally)
Same with in some crazy case the site is down for 7+ days